modules/layers/src/path-layer/path-layer-vertex.glsl.js

```diff
@@ -68,8 +68,17 @@ vec3 lineJoin(
   float sideOfPath = positions.y;
   float isJoint = float(sideOfPath == 0.0);
 
-  vec2 deltaA = (currPoint.xy - prevPoint.xy) / width;
-  vec2 deltaB = (nextPoint.xy - currPoint.xy) / width;
+  vec3 deltaA3 = (currPoint - prevPoint);
+  vec3 deltaB3 = (nextPoint - currPoint);
+
+  mat3 rotationMatrix;
+  if (project_uProjectionMode == PROJECTION_MODE_GLOBE && !billboard) {
```

Comment:
I wonder if we could have something like a `project_needsRotation()` method so we don't need to look at the values of module uniforms?